### PR TITLE
Remove UserHistory Feature and PageViews from Algolia 

### PR DESCRIPTION
**Pull request by mstruve**
## What type of PR is this? (check all applicable)
- [x] Refactor

## Description
This feature is currently not in use so rather than spend a bunch of time moving something over to Elasticsearch that we are not using, I think it would be easier to remove it. Then when we are ready to actually implement it we can do it with our new set of tools and exactly the way we want.

And FWIW, the boss said this was OK 😃
Screen Shot 2020-02-17 at 2 43 52 PM

## Related Tickets & Documents
https://github.com/thepracticaldev/dev.to/projects/6#card-32955480

## Added tests?
- [x] no, because they aren't needed

## Added to documentation?
- [x] no documentation needed

alt_text

### CODE REVIEW

**Comment by rhymes** [comment]
I use it ahhahh 😢

**Comment by benhalpern** [comment]
> I use it ahhahh 😢

I think a less-rushed re-implementation could be in order. Part of me wants to not scrap it entirely and instead re-implement right now with plain-old ActiveRecord.

But I think removing code, and revisiting this once we're truly stable and aligned to how the whole thing plays out is probably the most disciplined thing.

File: config/initializers/datadog_apm.rb [hunk 1]
```diff
@@ -6,6 +6,7 @@
   c.tracer enabled: Rails.env.production?
   c.tracer partial_flush: true
   c.tracer priority_sampling: true
+  c.use :elasticsearch
```
Comment (nickytonline) [inline config/initializers/datadog_apm.rb]: This APM setting does not appear related to the changes in this PR unless it's me missing something which is highly possible ;)

Comment (mstruve) [reply config/initializers/datadog_apm.rb]: GDI! 🤦‍♀

**Comment by rhymes** [comment]
@mstruve @benhalpern I perfectly understand, let's remove code! We can always dig it out from git history if we need to, or not :)
